### PR TITLE
edited the reference in the introduction

### PR DESCRIPTION
**Pull request by moktansang**

### CODE REVIEW

File: SRS.md [hunk 1]
```diff
@@ -20,16 +20,13 @@ The best way to decrease expenses and increase profit in the restaurant business
 The proposed system will increase order accuracy and decrease the time between the customer sitting down and receiving their food. It will also decrease the time the customer has to wait to pay for their order. This should increase the overall satisfaction of the customer, promoting return business.
 
 ##### 1.2 - Scope
-To start, we are going to focus only on the customers, the servers, and the kitchen. By providing a middle-system, we will be able to speed up the 
+To start, we are going to focus only on the customers, the servers, and the kitchen. By acting as an intermediate part, we will be able to speed up the 
 process of ordering food, getting the order to the kitchen, and getting completed orders to tables. Not only will this system benefit the customer, 
-but servers will now be able to manage their tables and orders more efficiently.
+but servers will now be able to manage their tables and orders more efficiently. The app will be used for several business promotion and customer can quickly give their recommendations to their dear ones.
```
Comment (jsainz237) [inline SRS.md]: Not sure if we need this sentence. We don't really mention this anywhere else.

File: SRS.md [hunk 2]
```diff
@@ -20,16 +20,13 @@ The best way to decrease expenses and increase profit in the restaurant business
 The proposed system will increase order accuracy and decrease the time between the customer sitting down and receiving their food. It will also decrease the time the customer has to wait to pay for their order. This should increase the overall satisfaction of the customer, promoting return business.
 
 ##### 1.2 - Scope
-To start, we are going to focus only on the customers, the servers, and the kitchen. By providing a middle-system, we will be able to speed up the 
+To start, we are going to focus only on the customers, the servers, and the kitchen. By acting as an intermediate part, we will be able to speed up the
```
Comment (jsainz237) [inline SRS.md]: maybe something like "...acting as an intermediate part between the restaurant staff and the customer..." might sound a bit more clear?

What do you think?

File: SRS.md (hunk 1 above)
Comment (moktansang) [reply SRS.md]: ok i will remove it them

File: SRS.md (hunk 2 above)
Comment (moktansang) [reply SRS.md]: absolutely

**Pull request closed without merging**